targets/ingester/src/lib/getJson.js

```diff
@@ -0,0 +1,17 @@
+import { readFile } from "fs/promises";
+import path from "path";
+import { fileURLToPath } from "url";
+
+// mjs equivalent for __dirname
+// https://stackoverflow.com/a/50052194
+const _dirname = path.dirname(fileURLToPath(import.meta.url));
+
+/**
+ *
+ * @param {string} file
+ */
+export async function getJson(file) {
+  const filePath = path.join(_dirname, "..", "..", "data", ...file.split("/"));
```

Comment:
wondering: `file` is not enough instead of `...file.split("/")` ?

Reply:
to remove all `/`separator that not work on windows ?

Reply:
indeed,forgot zindozs